Tests/test_bool.py

```diff
@@ -90,8 +88,6 @@ class MyLong(long): pass
         cases = []
         cases += [(ClassWithBool(x), y) for x, y in bool_cases]
         cases += [(ClassWithLen(x), y) for x, y in len_cases]
-        cases += [(ClassWithLen(long(x)), y) for x, y in len_cases if isinstance(x, int)]
```

Comment:
No value in testing with `big(x)` and `myint(x)`?

Reply:
I suppose there is. There is already a test with a big int (to test overflow) but not a successful conversion.